packages/api/cms-api/src/user-permissions/dto/list-user.ts

```diff
@@ -0,0 +1,22 @@
+import { Field, ObjectType } from "@nestjs/graphql";
+
+import { PaginatedResponseFactory } from "../../common/pagination/paginated-response.factory";
+import { User } from "./user";
+
+@ObjectType()
+class UserPermissionsListUser extends User {
+    @Field()
+    countPermissions: number;
+
+    @Field()
+    countContentScopes: number;
```

Comment:
Could this be field resolvers for the `User` object type? Does it make sense to have these somewhere else?